test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -219,21 +218,14 @@ public void close() throws IOException {
                                     System.setProperty(entry.getKey(), val);
                                 }
                             }
-                            tm.close();
                         }
                     }
                 }
             };
             Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
                 @Override
                 public void run() {
-                    try {
-                        shutdownTask.close();
```

Comment:
Removed because the `shutdownTask` gets closed as part of JUnit 5's closing of the ExtensionState.